src/components/app/ListOfPublishedProfiles.js

```diff
@@ -114,7 +114,7 @@ class PublishedProfile extends React.PureComponent<
     if (!urlPath.startsWith('/')) {
       urlPath = '/' + urlPath;
     }
-    const location = `${window.location.origin}/${urlPath}`;
+    const location = `${window.location.origin}${urlPath}`;
```

Comment:
The `/` is already added by the 3 previous lines. As a result we had a duplicate `/` when loading the page. This wasn't a big deal, just was a bit inelegant.